src/motion/utils/valid-prop.ts

```diff
@@ -53,6 +53,9 @@ const validMotionProps = new Set<keyof MotionProps>([
     "whileDrag",
     "onHoverEnd",
     "onHoverStart",
+    "whileFocus",
```

Comment:
There'll be a merge conflict here where I've started to spread the gestureHandlers straight into this Set. You'll be able to safely delete these lines.